modules/shadertools/src/modules/picking/picking.js

```diff
@@ -48,44 +41,65 @@ const float COLOR_SCALE = 1. / 255.;
 bool isVertexPicked(vec3 vertexColor) {
   return
     picking_uSelectedColorValid &&
-    abs(vertexColor.r - picking_uSelectedColor.r) < picking_uThreshold &&
-    abs(vertexColor.g - picking_uSelectedColor.g) < picking_uThreshold &&
-    abs(vertexColor.b - picking_uSelectedColor.b) < picking_uThreshold;
```

Comment:
Probably not used much, but this is a breaking change?

Reply:
This is not used anywhere in luma or deck.

Reply:
Are we changing the behavior, i.e. isn't it `picking_uThreshold` 1.0 by default, and now it is `0.001`, should look out for any regressions if so and make a note in the code.

Reply:
What use case does this break? `0.001` works for both normalized and non-normalized colors.